components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -205,25 +212,33 @@ const ContributorHighlightCard = ({
   const handleUpdateHighlight = async (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
 
-    const isValidGhUrl = highlight.prLink.match(/((https?:\/\/)?(www\.)?github\.com\/[^\/]+\/[^\/]+\/pull\/[0-9]+)/);
+    const pullrequestLink = highlight.prLink.trim();
     if (wordCount > wordLimit) {
       setError("Character limit exceeded");
       return;
     }
 
-    if (isValidGhUrl) {
+    if (isValidPullRequestUrl(pullrequestLink) || isValidIssueUrl(pullrequestLink)) {
       const { apiPaths } = generateApiPrUrl(highlight.prLink);
       const { repoName, orgName, issueId } = apiPaths;
       setLoading(true);
-      const res = await fetchGithubPRInfo(orgName, repoName, issueId);
+      const res = highlight.prLink.includes("issues")
```

Comment:
Make the check reusable for down below also

```suggestion
      const isIssue = highlight.prLink.includes("issues")
      const res = isIssue
```